### ISSUE TITLE
game crashes as soon as i try to skill any ability

### ISSUE THREAD

**Issue by bitfade** [ValveSoftware/Dota-2#818]
linux client, create a lobby, pick witch doctor, try to skill any ability, game crashes.

just got low prio because i couldn't even reconnect after one of my team mates skilled one of my abilities when i was disconnected.

it all started after the latest update, was fine before it. I can attach core dump if needed

**Comment by Tele42** [comment]
Please include your system information as requested at https://github.com/ValveSoftware/Dota-2#system-information, especially any CrashIDs related to the issue.

**Comment by bitfade** [comment]
Computer Information:
    Manufacturer:  Unknown
    Model:  Unknown
    Form Factor: Desktop
    No Touch Input Detected

Processor Information:
    CPU Vendor:  GenuineIntel
    CPU Family:  0x6
    CPU Model:  0x3a
    CPU Stepping:  0x9
    CPU Type:  0x0
    Speed:  3800 Mhz
    4 logical processors
    4 physical processors
    HyperThreading:  Unsupported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Unsupported
    SSE41:  Supported
    SSE42:  Supported
    AES:  Supported
    AVX:  Supported
    CMPXCHG16B:  Supported
    LAHF/SAHF:  Supported
    PrefetchW:  Unsupported

Network Information:
    Network Speed:  

Operating System Version:
    Ubuntu 14.04.4 LTS (64 bit)
    Kernel Name:  Linux
    Kernel Version:  4.3.5-040305-lowlatency
    X Server Vendor:  The X.Org Foundation
    X Server Release:  11501000
    X Window Manager:  KWin
    Steam Runtime Version:  steam-runtime-release_2016-02-23

Video Card:
    Driver:  NVIDIA Corporation GeForce GTX 970/PCIe/SSE2

```
Driver Version:  4.5.0 NVIDIA 352.63
OpenGL Version: 4.5
Desktop Color Depth: 24 bits per pixel
Monitor Refresh Rate: 59 Hz
VendorID:  0x10de
DeviceID:  0x13c2
Number of Monitors:  1
Number of Logical Video Cards:  1
Primary Display Resolution:  2560 x 1440
Desktop Resolution: 2560 x 1440
Primary Display Size: 23,50" x 13,23"  (26,97" diag)
                                        59,7cm x 33,6cm  (68,5cm diag)
Primary Bus: PCI Express 16x
Primary VRAM: 4096 MB
Supported MSAA Modes:  2x 4x 8x 16x 
```

Sound card:
    Audio device: Realtek ALC887-VD

Memory:
    RAM:  16009 Mb

Miscellaneous:
    UI Language:  English
    LANG:  it_IT.UTF-8
    Microphone:  Not set
    Total Hard Disk Space Available:  112546 Mb
    Largest Free Hard Disk Block:  66005 Mb
    VR Headset: None detected

```
Wed Mar 30 00:17:09 2016 GMT: file ''/tmp/dumps/crash_20160330021301_1.dmp'', upload yes: ''Discarded=1''
Wed Mar 30 00:19:17 2016 GMT: file ''/tmp/dumps/crash_20160330021708_1.dmp'', upload yes: ''Discarded=1''
Wed Mar 30 00:36:52 2016 GMT: file ''/tmp/dumps/crash_20160330021915_1.dmp'', upload yes: ''Discarded=1''
Wed Mar 30 00:37:20 2016 GMT: file ''/tmp/dumps/crash_20160330023651_1.dmp'', upload yes: ''Discarded=1''
Wed Mar 30 00:37:48 2016 GMT: file ''/tmp/dumps/crash_20160330023719_1.dmp'', upload yes: ''Discarded=1''
Wed Mar 30 00:38:23 2016 GMT: file ''/tmp/dumps/crash_20160330023753_1.dmp'', upload yes: ''Discarded=1''
Wed Mar 30 00:39:10 2016 GMT: file ''/tmp/dumps/crash_20160330023823_1.dmp'', upload yes: ''Discarded=1''
Wed Mar 30 00:41:19 2016 GMT: file ''/tmp/dumps/crash_20160330023909_1.dmp'', upload yes: ''CrashID=bp-84962b39-4d74-4e4b-8b17-511a32160329''
Wed Mar 30 00:42:40 2016 GMT: file ''/tmp/dumps/crash_20160330024211_1.dmp'', upload yes: ''CrashID=bp-ba4b9aa5-dbcb-4527-b115-efe0f2160329''
Wed Mar 30 00:43:05 2016 GMT: file ''/tmp/dumps/crash_20160330024238_1.dmp'', upload yes: ''Discarded=1''
Wed Mar 30 00:43:48 2016 GMT: file ''/tmp/dumps/crash_20160330024303_1.dmp'', upload yes: ''Discarded=1''
Wed Mar 30 00:44:15 2016 GMT: file ''/tmp/dumps/crash_20160330024347_1.dmp'', upload yes: ''Discarded=1''
Wed Mar 30 00:45:41 2016 GMT: file ''/tmp/dumps/crash_20160330024509_1.dmp'', upload yes: ''CrashID=bp-f97178a4-04de-4574-a0cb-16f3e2160329''
Wed Mar 30 00:46:15 2016 GMT: file ''/tmp/dumps/crash_20160330024540_1.dmp'', upload yes: ''Discarded=1''
Wed Mar 30 00:55:49 2016 GMT: file ''/tmp/dumps/crash_20160330025437_1.dmp'', upload yes: ''Discarded=1''
Wed Mar 30 00:57:43 2016 GMT: file ''/tmp/dumps/crash_20160330025649_1.dmp'', upload yes: ''Discarded=1''
```

Edit - CrashID related to the issue
CrashID=bp-dee5819a-deee-4804-8646-8da462160329

Edit 2 - Seems like the new update fixed it .... thx for the low prio....

**Comment by glajchs** [comment]
While it isn't exactly what the title of this bug describes, I get the feeling that I was seeing the same issue.  I just finished a ~50 minute game where, at the beginning I could use the courier, but as of about 10 minutes into the game my client started crashing.  Near the end I realized that it was anytime I selected the courier (usually by hitting "2" for the hotkey group).  I was able to confirm this in a solo bot game as well.

I just downloaded the DOTA patch from about 5 minutes ago, and now I can select the courier in the solo bot match, so I'm hoping/assuming that the patch fixed my issue as well.

Thought I'd add my story to the pool.

System specs here:
https://gist.github.com/glajchs/38d32eb4ff5c434727a5b3982358a94e
